src/components/button/button.stories.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select, boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import notes from './notes.md';
+import Button from './button';
+
+const clickAction = action('click');
+
+storiesOf('Button', module)
```

Comment:
The text at the top of this story doesn't have the best formatting, and isn't much use in its current form.

![image](https://user-images.githubusercontent.com/26925206/52704400-647ceb00-2f78-11e9-8b96-1740e2a2c7ad.png)

Reply:
Sure. I need to investigate it how to remove it.
I am wondering if we should fork info addon and try to fit it.

Reply:
Using the following CSS will fix it `white-space: pre-wrap;` is there a way to use that?